### PR TITLE
feat: use github-comment-manager for PR commenting

### PR DESCRIPTION
**Pull request by MarkConway94**
... to inside packages/test-helpers/src so it transpiles

### CODE REVIEW

File: packages/test-helpers/src/bin/github-comment-manager.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  const comments = await getExpoComments(
+    account,
+    token,
+    pullRequest,
+    repository
+  );
+  return deleteCommentsFromList(comments, account, token, repository);
+};
+
+module.exports = {
```
Comment (L0wry) [inline packages/test-helpers/src/bin/github-comment-manager.js]: as this is es6 now i think we can do export default

File: packages/test-helpers/src/bin/run.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env node
 
+import githubCommentManager from "./github-comment-manager";
+
 const program = require("commander");
```
Comment (L0wry) [inline packages/test-helpers/src/bin/run.js]: should be able to import the requires now that we are transpiling

last thing promise!

File: packages/test-helpers/src/bin/github-comment-manager.test.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
+/* global describe it expect */
```
Comment (asifhafeez) [inline packages/test-helpers/src/bin/github-comment-manager.test.js]: what does this comment mean?

File: packages/test-helpers/src/bin/github-comment-manager.test.js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    id: 2,
+    body: "If you use Expo, view our components by scanning this qr code:",
+    user: {
+      login: account
+    }
+  }
+];
+
+describe("deleteAllExpoComments", () => {
+  it("should not delete any comments", async () => {
```
Comment (asifhafeez) [inline packages/test-helpers/src/bin/github-comment-manager.test.js]: Why would it not delete any comments? This test name needs to be more descriptive.

Comment (asifhafeez) [inline packages/test-helpers/src/bin/github-comment-manager.test.js]: Why would it not delete any comments? This test name needs to be more descriptive.

File: packages/test-helpers/src/bin/github-comment-manager.test.js (hunk 3 above)
Comment (asifhafeez) [inline packages/test-helpers/src/bin/github-comment-manager.test.js]: what does this comment mean?

Comment (MarkConway94) [reply packages/test-helpers/src/bin/github-comment-manager.test.js]: Its used by lint to set global variables

File: packages/test-helpers/src/bin/github-comment-manager.test.js (hunk 4 above)
Comment (MarkConway94) [reply packages/test-helpers/src/bin/github-comment-manager.test.js]: It shouldn't delete any comments as the filter should not find any comments made by the specified account. Will make it more clear.

**Pull request merged**